specification/machinelearningservices/data-plane/Microsoft.MachineLearningServices/preview/2019-05-01/execution.json

```diff
@@ -0,0 +1,1267 @@
+          "description": "The path on the compute target.",
+          "type": "string"
+        },
+        "overwrite": {
+          "description": "Whether to overwrite the data if existing.",
+          "type": "boolean",
+          "example": "False"
+        }
+      }
+    },
+    "EnvironmentDefinition": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "The name of the environment.",
+          "type": "string",
+          "example": "mydevenvironment"
+        },
+        "version": {
+          "description": "The environment version.",
+          "type": "string",
+          "example": "1"
+        },
+        "python": {
+          "$ref": "#/definitions/PythonSection",
+          "description": "Settings for a Python environment."
+        },
+        "environmentVariables": {
+          "description": "Definition of environment variables to be defined in the environment.",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "docker": {
+          "$ref": "#/definitions/DockerSection",
+          "description": "The definition of a Docker container."
+        },
+        "spark": {
+          "$ref": "#/definitions/SparkSection",
+          "description": "The configuration for a Spark environment."
+        }
+      }
+    },
+    "HistoryConfiguration": {
+      "type": "object",
+      "properties": {
+        "outputCollection": {
+          "description": "Set to true to collect outputs and store in run history.",
+          "type": "boolean",
+          "example": "True"
+        },
+        "directoriesToWatch": {
+          "description": "The list of directories to monitor and upload files from.",
+          "uniqueItems": false,
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "example": "[\"logs\", \"outputs\"]"
+        }
+      },
+      "additionalProperties": {
+        "type": "object"
+      }
+    },
+    "SparkConfiguration": {
+      "type": "object",
+      "properties": {
+        "configuration": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "AMLComputeConfiguration": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "The name of the AMLCompute cluster.",
+          "type": "string",
+          "example": "my8nodeCluster"
+        },
+        "vmSize": {
+          "description": "VM size of the Cluster to be created. Allowed values are Azure vm sizes.\r\nThe list of vm sizes is available in https://docs.microsoft.com/en-us/azure/cloud-services/cloud-services-sizes-specs",
+          "type": "string",
+          "example": "Standard_D2_v2"
+        },
+        "vmPriority": {
+          "description": "VM priority of the Cluster to be created. Allowed values are dedicated and lowpriority.",
```

Comment:
if there are only 2 allowed values, please consider an enum

Reply:
currently in PR with service owners

Reply:
fixed